packages/jazz-run/src/startSync.ts

```diff
@@ -0,0 +1,121 @@
+import { Command, Options } from "@effect/cli";
+import {
+    ControlledAgent,
+    LocalNode,
+    WasmCrypto,
+    cojsonInternals,
+} from "cojson";
+import { WebSocketServer } from "ws";
+
+import { createWebSocketPeer } from "cojson-transport-ws";
+import { Effect } from "effect";
+import { SQLiteStorage } from "cojson-storage-sqlite";
+import { dirname } from "node:path";
+import { mkdir } from "node:fs/promises";
+
+const port = Options.text("port")
+    .pipe(Options.withAlias("p"))
+    .pipe(
+        Options.withDescription(
+            "Select a different port for the WebSocket server. Default is 4200",
+        ),
+    )
+    .pipe(Options.withDefault("4200"));
+
+const inMemory = Options.boolean("in-memory").pipe(
+    Options.withDescription("Use an in-memory storage instead of file-based"),
+);
+
+const db = Options.file("db")
+    .pipe(
+        Options.withDescription(
+            "The path to the file where to store the data. Default is 'sync-db/storage.db'",
+        ),
+    )
+    .pipe(Options.withDefault("sync-db/storage.db"));
+
+export const startSync = Command.make(
+    "sync",
+    { port, inMemory, db },
+    ({ port, inMemory, db }) => {
+        return Effect.gen(function* () {
+            const crypto = yield* Effect.promise(() => WasmCrypto.create());
+
+            const wss = new WebSocketServer({ port: parseInt(port) });
+
+            console.log(
+                "COJSON sync server listening on port " + wss.options.port,
+            );
+
+            /**
+             * In Jazz the communication is decentralized
+             * so the sync server is implemented as "just another peer" in the network
+             *
+             * LocalNode is the class we use to manage the connected peers that could be:
+             * - the storage layer
+             * - the WebSocket connections
+             */
```

Comment:
@aeplay 

Could you double-check if what I'm saying here is correct?

Reply:
I'd say this kind of stuff belongs into the internal documentation of LocalNode and high level docs of CoJSON as a protocol - and if it's clear there the use of websocket and storage peers here is obvious, so let's remove the comment here for now

Reply:
(your understanding is correct though)